### ISSUE TITLE
Integrated test wiki

### ISSUE THREAD

**Issue by FarRoss** [GoogleCloudPlatform/iot-device-sdk-embedded-c#107]
After compiling using this command ```make TARGET=linux-static-debug CONFIG=posix_fs-posix_platform-mqtt_localhost```, how can I run an example to connect to my Local MQTT broker? 

Are there any wiki documents with a sample example?

**Comment by atigyi** [comment]
With this build the client will create an insecure TCP socket connecting to `localhost:1883`.

For port 1883, see [this part of the config](https://github.com/GoogleCloudPlatform/iot-device-sdk-embedded-c/blob/master/make/mt-config/mt-config.mk#L174-L189) and [this](https://github.com/GoogleCloudPlatform/iot-device-sdk-embedded-c/blob/master/src/libiotc/iotc_config.h#L45-L48).

This client supports solely the Google Cloud IoT MQTT Bridge authentication process ([some client side hints](https://cloud.google.com/iot/docs/how-tos/mqtt-bridge?hl=en#configuring_mqtt_clients)). Thus the target (localhosted) MQTT Broker should accept this auth flow. First best to be a "happy" broker, accepting every connection. 

Basically, [the posix example](https://github.com/GoogleCloudPlatform/iot-device-sdk-embedded-c/tree/master/examples/iot_core_mqtt_client) should work, just build and run it.
